pkg/cnab/extensions/dependencies_test.go

```diff
@@ -21,21 +20,27 @@ func TestReadDependencyProperties(t *testing.T) {
 	deps, err := ReadDependencies(*bun)
 
 	assert.NotNil(t, deps, "Dependencies was not populated")
-	assert.Len(t, deps.Requires, 2, "Dependencies.Requires is the wrong length")
-
-	dep := deps.Requires["storage"]
-	assert.NotNil(t, dep, "expected Dependencies.Requires to have an entry for 'storage'")
-	assert.Equal(t, "somecloud/blob-storage", dep.Bundle, "Dependency.Bundle is incorrect")
-	assert.Nil(t, dep.Version, "Dependency.Version should be nil")
-
-	dep = deps.Requires["mysql"]
-	assert.NotNil(t, dep, "expected Dependencies.Requires to have an entry for 'mysql'")
-	assert.Equal(t, "somecloud/mysql", dep.Bundle, "Dependency.Bundle is incorrect")
-	assert.True(t, dep.Version.AllowPrereleases, "Dependency.Bundle.Version.AllowPrereleases should be true")
-	assert.Equal(t, []string{"5.7.x"}, dep.Version.Ranges, "Dependency.Bundle.Version.Ranges is incorrect")
+	assert.Len(t, deps, 2, "Dependencies.Requires is the wrong length")
+
+	for _, dep := range deps {
+		if dep.Name == "storage" {
+
+			assert.NotNil(t, dep, "expected Dependencies.Requires to have an entry for 'storage'")
```

Comment:
Now that this has been changed to ranging over deps, checking for nil here doesn't apply. If it were nil we would have gotten an error on the previous line.